app/game/game.controller.js

```diff
@@ -643,6 +643,23 @@ module.exports = (db, Sentry) => {
                     error: 'Something went wrong.'
                 });
             }
+        },
+        getPlayerStatsByWeek: async (req, res) => {
+            try {
+                if (!req.query.week || !req.query.athleteId) {
+                    res.status(400).send({
+                        error: 'athletId and week must be specified'
+                    });
+                } else {
+                    const data = await service.getPlayerStatsByWeek(req.query.week, req.query.athleteId);

```

Comment:
Should require a season parameter on this since this would potentially return multiple season worth of data.